src/olympia/discovery/tests/test_admin.py

```diff
@@ -301,7 +301,7 @@ def test_can_add_primary_hero_with_discovery_edit_permission(self):
         assert item.custom_addon_name == 'Xäxâxàxaxaxa !'
         assert item.recommendable is True
         hero = PrimaryHero.objects.last()
-        assert hero.image == 'Ubo@2x.jpg'
+        assert hero.select_image == 'Ubo@2x.jpg'
```

Comment:
and assert here that `hero.select_image_pk == <the latest PrimaryHeroImage object>`.
Something like `assert hero.select_image_pk == image.pk` and _before_ the request is made you've done `image = PrimaryHeroImage.objects.create(custom_image='foo.jpg')` and then the `'primaryhero-0-select_image':` above is referencing the PrimaryHeroImage object.